modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -15,10 +15,10 @@ The Red Hat Telco Radio Access Network (RAN) version {product-version} solution
 |Software version
 
 |Hub cluster {product-title} version
-|4.12
+|4.13
 
 |{ztp} plugin
-|4.10, 4.11, or 4.12
+|4.11, 4.12, or 4.13
 
 |{rh-rhacm-first}
 |2.6, 2.7
```

Comment:
We use ACM 2.7 for 4.13 testing. 

And we need to add ACM 2.8 after ACM 2.8 GA.